en/SUMMARY.md

```diff
@@ -228,6 +228,7 @@
       * [Freefly RTK GPS](gps_compass/rtk_gps_freefly.md)
       * [Holybro H-RTK-F9P](gps_compass/rtk_gps_holybro_h-rtk-f9p.md)
       * [Holybro H-RTK-M8P](gps_compass/rtk_gps_holybro_h-rtk-m8p.md)
+      * [LOCOSYS Hawk A1 GNSS](gps_compass/gps_locosys_hawk_a1.md)
```

Comment:
If you don't have this, it won't appear in the sidebar

Reply:
Also yes, I always miss this, :/